docs/index.njk

```diff
@@ -209,7 +209,7 @@ title: Home
           </div>
           <div class="col-10 col-md-6 col-lg-5 text-center text-lg-end">
               <a href="/get-started/" class="btn btn-white btn-lg">
-                Get started<span class="fas fa-arrow-right ms-1" aria-hidden="true"></span>
+                Get Started<span class="fas fa-arrow-right ms-1" aria-hidden="true"></span>
```

Comment:
earlier comment about not capitalizing all words in a button, but i will defer to @rltucker

Reply:
Other non-document uses of this include 404 page, home page, and the sitemap page.

Reply:
I cap all words in a button (except for words like a, an, the when they appear no as first word.